src/routes/routes.ts

```diff
@@ -132,3 +132,9 @@ export const demoSafeRoute = generateSafeRoute(SAFE_ROUTES.APPS, {
   shortName: 'eth',
   safeAddress: DEMO_SAFE_MAINNET,
 })
+
+export const getShareSafeAppUrl = (appUrl: string, chainId: string): string => {
```

Comment:
Can this be used elsewhere in the Safe? I'd add a few tests for it as well.

I'm not sure if it belongs in this file. What do you think?

Reply:
I think the function can stay but the demo Safe address should move to `src/utils/constants` and the generated route in the relevant component. I assume it is only used in one?

Reply:
yep, but that was addressed in the other PR